### PR TITLE
Add Farcaster icon to footer

### LINKED ISSUE #9576: Add farcaster icon to footer
Link to https://warpcast.com/~/channel/koda

### PR DESCRIPTION
**Pull request by wangyangjun**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #9576 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/ksm/transfer?target=168JVWrm3qa8Q1pf5Tqddxwkkwa3yiKqNMwvnD6g53W3tsmU&usdamount=100&donation=true)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

Screenshot 2024-02-28 at 22 51 03

### CODE REVIEW

File: components/TheFooter.vue [hunk 1]
```diff
@@ -119,6 +119,24 @@
...
+                height="20"
+                viewBox="0 0 1000 800"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M257.778 155.556H742.222V844.445H671.111V528.889H670.414C662.554 441.677 589.258 373.333 500 373.333C410.742 373.333 337.446 441.677 329.586 528.889H328.889V844.445H257.778V155.556Z"
+                  fill="white" />
+                <path
+                  d="M128.889 253.333L157.778 351.111H182.222V746.667C169.949 746.667 160 756.616 160 768.889V795.556H155.556C143.283 795.556 133.333 805.505 133.333 817.778V844.445H382.222V817.778C382.222 805.505 372.273 795.556 360 795.556H355.556V768.889C355.556 756.616 345.606 746.667 333.333 746.667H306.667V253.333H128.889Z"
+                  fill="white" />
```
Comment (Jarsen136) [inline components/TheFooter.vue:135]: Could you pls move the inline-svg to another path as a single file?
(public/icons/xxx.svg)

Comment (wangyangjun) [reply components/TheFooter.vue]: updated as you suggested

**Comment by Jarsen136** [comment]
icon at light mode
image

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
@wangyangjun please generate dot address and put it into your PR template description and ping me on telegram

**Pull request merged**